debounce.js

```diff
@@ -2,10 +2,11 @@ function debounce(fn, delay) {
   let timeID = null;
 
   return function (...args) {
+    let context = this;
```

Comment:
might be `const` instead of `let`. doesn't seem it can be changed

Reply:
if we define a variable as const, we will not be able to redefine it in the future

Reply:
where it can be redefined?

Reply:
in line 7 we will not be able to override the value. On line 7 we assign the value returned by setTimeout

Reply:
probably there's a confusion here. I'm talking about `context` variable not `timeID` (which should be redefined indeed).